src/bundle/Resources/views/LinkManager/view.html.twig

```diff
@@ -4,25 +4,22 @@
 
 {%- block breadcrumbs -%}
     {% include '@EzPlatformAdminUi/parts/breadcrumbs.html.twig' with { items: [
-        { value: 'breadcrumb.admin'|trans(domain='messages')|desc('Admin') },
         { value: 'url.list'|trans, url: path('admin_link_manager_list') },
-        { value: 'url.view'|trans({ '%id%': url.id }) }
+        { value: 'url.view'|trans({ '%url%': url.url|length > 60 ? url.url|slice(0, 60) ~ '...' : url.url }) }
     ]} %}
 {%- endblock -%}
 
 {%- block pageTitle -%}
     {% include '@EzPlatformAdminUi/parts/page_title.html.twig' with {
-        title: 'url.view'|trans({ '%id%': url.id }),
+        title: 'url.view'|trans({ '%url%': url.url|length > 60 ? url.url|slice(0, 60) ~ '...' : url.url }),
```

Comment:
Self-nitpick: We can use `truncate` filter instead of reinventing the wheel: https://github.com/twigphp/Twig-extensions/blob/master/doc/text.rst